.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+Please include a summary of the change, motivation and context
+
+- **on a feature**: describe the feature and how this change fits in it, e.g. this PR makes kafka message.max.bytes configurable to better support batching
+- **on a refactor**: describe why this is better than previous situation e.g. this PR changes logic for retry on healthchecks to avoid false positives
+- **on a bugfix**: link relevant information about the bug (github issue or slack thread) and how this change solves it e.g. this change fixes #99999 by adding a lock on read/write to avoid data races.
+
+### How has this been tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Docker-compose
+- [ ] Helm install
+
+**Test Configuration**:
```

Comment:
This is not needed, the ultimate testing authority is our CI. We literally don't care if the tests pass on the author's machine.